test/models/asyncapi_test.js

```diff
@@ -126,6 +137,17 @@ describe('AsyncAPIDocument', () => {
     });
   });
 
+  describe('#hasParameters()', function () {
+    it('should return a boolean indicating if the AsyncAPI document has channel parameters', () => {
+      const doc = { channels: { test1: { parameters: { test1param: { description: "test1param" } } } } };
+      const docNoChannelParams = { channels: { test: { description: "test" } } };
+      const d = new AsyncAPIDocument(doc);
+      const d2 = new AsyncAPIDocument(docNoChannelParams);
+      expect(d.channel('test1').hasParameters()).to.equal(true);
+      expect(d2.channel('test').hasParameters()).to.equal(false);
+    });
+  });
+
```

Comment:
I think this test should be here: https://github.com/asyncapi/parser-js/blob/master/test/models/channel_test.js